protobuf-c/protobuf-c.c

```diff
@@ -2099,7 +2099,7 @@ typedef struct _ScannedMember ScannedMember;
 /** Field as it's being read. */
 struct _ScannedMember {
 	uint32_t tag;              /**< Field tag. */
-	uint8_t wire_type;         /**< Field type. */
+	ProtobufCWireType wire_type; /**< Field type. */
```

Comment:
This is not good, we should not be changing the size of `ScannedMember` while fixing compiler warnings.